src/components/SecondaryPanes/Frames/Frame.js

```diff
@@ -36,7 +36,12 @@ function FrameLocation({ frame }: FrameLocationProps) {
     );
   }
 
-  const filename = getFilename(frame.source);
+  let filename = getFilename(frame.source);
+  // The call stack panel space is limited so remove the query strings
+  const qMarkIdx = filename.indexOf("?");
+  if (qMarkIdx > 0) {
+    filename = filename.slice(0, qMarkIdx);
+  }
```

Comment:
can we put this logic in `getFilename` and update the tests

Reply:
@jasonLaster updated thanks